### ISSUE TITLE
Avoid mounting GCP service account into the operator's pod

### ISSUE THREAD

**Issue by ljakab** [CloudNativeSDWAN/cnwan-operator#45]
Once PR #43 is merged, adopt the new code to get GCP service account from the Kubernetes secret, and avoid mounting it into the pod.

**Comment by SunSince90** [comment]
Hi @ljakab, this is duplicate of #32 and will probably come with its own PR

**Comment by ljakab** [comment]
Indeed, I didn't realize there was already an issue open on this. While doing the review for the cloud metadata exposure work, I didn't want us to forget this, so I quickly wrote it down.

**Comment by ljakab** [comment]
Duplicate of #32
